### PR TITLE
Add useJunitPlatform() to build.gradle

### PR DESCRIPTION
**Pull request by allredj**
This is required for the JUnit tests to run.

Source: https://docs.gradle.org/current/userguide/java_testing.html#using_junit5

Issue: https://github.com/halo-dev/halo/issues/1127

### CODE REVIEW

**Comment by allredj** [comment]
**Note to the maintainers**
The failing tests will need to be fixed before you can merge this.

**Comment by JohnNiang** [comment]
We will fix the failing tests soon.

**Comment by JohnNiang** [comment]
You have to merge halo/master branch into your `add-junit-to-build-gradle` branch, and push them up.

**Review by JohnNiang — APPROVED** [review]

**Comment by JohnNiang** [comment]
Thank you for your exploration about gradle test. It's my fault to introduce this issue.

**Pull request merged**
